sonar-python-plugin/src/main/java/org/sonar/plugins/python/indexer/SonarLintPythonIndexer.java

```diff
@@ -51,6 +53,13 @@ public void buildOnce(SensorContext context) {
     }
     this.projectBaseDir = context.fileSystem().baseDir();
     List<InputFile> files = getInputFiles(moduleFileSystem);
+    long nLines = files.stream().map(InputFile::lines).mapToLong(Integer::longValue).sum();
+    long maxLinesForIndexing = context.config().getLong(MAX_LINES_PROPERTY).orElse(DEFAULT_MAX_LINES_FOR_INDEXING);
+    if (nLines > maxLinesForIndexing) {
+      // Avoid performance issues for large projects
+      LOG.debug("Project symbol table deactivated due to project size");
```

Comment:
I think we can add info about `maxLinesForIndexing` and `nLines` in the debug message.